Cypress default config & README updates (#893)
mattermost/mattermost-plugin-playbooks: README.md

  * An example command looks like: `/playbook test bulk-data 10 3 342 2`

> ## Running E2E Tests

Reviewer: ```suggestion
## Running E2E tests
```

> When running E2E tests, local `mattermost-server` configuration may be unexpectedly modified if values are different than a corresponding key default in either `on_prem_default_config.json` or `cloud_default_default_config.json`. This can be avoided by setting `CYPRESS_developerMode=true` when calling Cypress scripts. For example: `CYPRESS_developerMode=true npm run cypress:open`.

Reviewer: ```suggestion
When running E2E tests, the local `mattermost-server` configuration may be unexpectedly modified if the values for `on_prem_default_config.json` or `cloud_default_default_config.json` are different to the corresponding default key . This can be avoided by setting `CYPRESS_developerMode=true` when calling Cypress scripts. For example: `CYPRESS_developerMode=true npm run cypress:open`.
```

Author: I think this phrasing shouldn't have the "default" in it. My thinking is that the aforementioned JSON files are the test defaults, not the local server config, but this implies the latter.

how about:

When running E2E tests, the local `mattermost-server` configuration may be unexpectedly modified if either `on_prem_default_config.json` or `cloud_default_config.json` (depending on the server edition) has conflicting values for the the same keys. This can be avoided by setting `CYPRESS_developerMode=true` when calling Cypress scripts. For example: `CYPRESS_developerMode=true npm run cypress:open`.

Reviewer: Thank you for that insight - great suggestion! I've applied it. :)
